pkg/kubelet/kubelet.go

```diff
@@ -271,22 +271,23 @@ func makePodSourceConfig(kubeCfg *kubeletconfiginternal.KubeletConfiguration, ku
 
 	// source of all configuration
 	cfg := config.NewPodConfig(config.PodConfigNotificationIncremental, kubeDeps.Recorder)
+	ctx := context.Background()
```

Comment:
Sorry, missed this one. It needs to be TODO so that it's replaced with a proper context in the future.
```suggestion
	ctx := context.TODO()
```
/remove lgtm

Reply:
okay, I add a todo for it